src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -3737,25 +3797,14 @@ Not sure how to handle these:
     const auto& refseq = curr_pep_ids[0].getHits()[0].getSequence();
     for (const auto& pep : curr_pep_ids)
     {
-      if (pep.getHits().size() > 1)
+      if (pep.getHits()[0].getSequence() != refseq)
```

Comment:
we could revert that. this relaxation was not needed in the end. but exception is also not really necessary since we always take the best hit only anyway. if we communicate that..